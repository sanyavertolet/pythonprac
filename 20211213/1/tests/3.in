[3454, 34523, 787687, 345, 8678, 3, 46, 75, 8, 4, 35, 34, 5, 3, 756, 8769]
