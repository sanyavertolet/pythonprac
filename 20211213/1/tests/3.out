[3, 3, 4, 5, 8, 34, 35, 46, 75, 345, 756, 3454, 8678, 8769, 34523, 787687]
